packages/website/pages/packages.tsx

```diff
@@ -12,9 +12,15 @@ import AllPackagesNavContent from '../components/navigation/all-packages-nav-con
 
 import data from '../pages-list';
 // eslint-disable-next-line import/no-unresolved
-import meta from '../data/packages-data.json';
+import meta2 from '../data/packages-data.json';
 import PageTitle from '../components/page-title';
 
+import { PackageInfo, Metadata } from '../types';
+
+const meta: Metadata = meta2;
+
+// TODO: TSFix We are typing metadata here, but I'm sure there is a better way to do this
```

Comment:
Don't we have that special re-import file somewhere? You could move the typing into there.

Reply:
Not for metadata, only for pageData, which is completely different obv.

This tsfix is wrong though as I already lifted up the typing of this.